src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -4266,6 +4266,31 @@ public static void TestConvertToNullable()
             Assert.Equal(2, d(2));
         }
 
+        [Fact]
```

Comment:
I could not yet figure how to run tests in both modes. 
The problem is that interpretation is not a runtime switch, it requires rebuild. It is easy to run test in interpretation mode locally by setting an environment variable and forcing rebuild.

I will be porting these fixes to ProjectN branch where they will definitely run in interpreter mode, but yes, we need a way to run interpreter in CI.